json/upickle/src/main/scala/tapir/json/upickle/TapirJsonuPickle.scala

```diff
@@ -0,0 +1,28 @@
+package tapir.json.upickle
+
+import java.nio.charset.StandardCharsets
+import scala.util.{Try, Success, Failure}
+import tapir.Codec.JsonCodec
+import tapir.DecodeResult.{Error, Value}
+import tapir._
+import upickle.default.{ReadWriter => RW}
```

Comment:
Is this alias necessary? `ReadWriter` is not too long to read and doesn't seem like shadowing anything.

Reply:
You're quite right. Fixed